L1Trigger/Phase2L1ParticleFlow/plugins/L1SeedConePFJetProducer.cc

```diff
@@ -32,19 +32,21 @@ class L1SeedConePFJetProducer : public edm::global::EDProducer<> {
   void produce(edm::StreamID, edm::Event& iEvent, const edm::EventSetup& iSetup) const override;
   /// ///////////////// ///
 
-  const float coneSize;
-  const unsigned nJets;
-  const bool HW;
-  const bool debug;
-  const bool doCorrections;
+  float coneSize;
+  bool wideConeJet;
+  unsigned nJets;
+  bool HW;
+  bool debug;
+  bool doCorrections;
   L1SCJetEmu emulator;
   edm::EDGetTokenT<std::vector<l1t::PFCandidate>> l1PFToken;
   l1tpf::corrector corrector;
 
-  std::vector<l1t::PFJet> processEvent_SW(std::vector<edm::Ptr<l1t::PFCandidate>>& parts) const;
+  std::vector<l1t::PFJet> processEvent_SW(std::vector<edm::Ptr<l1t::PFCandidate>>& work) const;
```

Comment:
Can this be `parts` in order to be consistent with https://github.com/cms-sw/cmssw/blob/4903a04b5c61d16c6189c4e06fb48df9bb9b45d5/L1Trigger/Phase2L1ParticleFlow/plugins/L1SeedConePFJetProducer.cc#L48 and https://github.com/cms-sw/cmssw/blob/4903a04b5c61d16c6189c4e06fb48df9bb9b45d5/L1Trigger/Phase2L1ParticleFlow/plugins/L1SeedConePFJetProducer.cc#L102